translations/en.i18n.json

```diff
@@ -87,6 +87,7 @@
       "media_not_ready_error_message": "Media is processing, check back soon.",
       "geo_location_error_title": "Geo location unavailable",
       "geo_location_error_message": "This content is unavailable in your region.",
+      "ip_restricted_error_message": "This media has been restricted to certain IP addresses.",
```

Comment:
did you notify product that this string will need a translation ?

Reply:
yes